ADAL/tests/ADAcquireTokenTests.m

```diff
@@ -986,6 +1167,62 @@ - (void)testAcquireTokenUsingFRT
      }];
     
     TEST_WAIT;
+    
+    // verify telemetry output
+    // there should be 1 telemetry events recorded as aggregation flag is ON
+    XCTAssertEqual([receivedEvents count], 1);
+    
+    // the following properties are expected in an aggregrated event
+    XCTAssertEqual([self adGetPropertyCount:[receivedEvents firstObject]
```

Comment:
Pull out `[receivedEvents firstObject]` into a local variable and use that instead.